src/processors/file-handler.js

```diff
@@ -159,14 +149,12 @@ export async function readFileInBatches({file, fileCache = []}) {
 
 export function processFileData({content, fileCache}) {
   return new Promise((resolve, reject) => {
-    const {data, header} = content;
+    const {data} = content;
 
     let format;
     let processor;
-    if (isCsvRows(data)) {
-      format = DATASET_FORMATS.csv;
-      processor = processCsvData;
```

Comment:
Let's not delete this. kepler processCsvData does 2 things: 
1. type detection, specifically to find `time` `geo` as well as the exact time format `yyyy-hh-dd mm:ss` 
2. basic data cleaning. lots of times people have string values like `N/A` `Null` `N/` mixed up with numbers. kepler finds them and fill it in with `null` so type detect won't fail. 

For case1,  there is a fall back in reducer, if `format` if time  is note detect, keler will call type detection again. Not sure loaders.gl will handle case 2

Reply:
To add to Ib's point, once the dataset is processed via loaders.gl, the output format for CSV files and JSON Row objects becomes the same (an array of records), so the `isCsvRows` function doesn't get evaluated. Any ideas on how to tackle this?